### DISCUSSION TITLE
Can Azure Bulk Credits Be Applied to GitHub Copilot Billing via Azure Subscription?

### DISCUSSION THREAD

**Discussion by khakie** [community/community#137659] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I have a question regarding billing for GitHub Copilot through an Azure Subscription.
If an organization is registered with ISV Hub as a Microsoft Partner and holds Azure free credits, can these credits be applied towards the billing for GitHub Copilot? Or is GitHub Copilot billed separately as an independent subscription?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @khakie - thanks for asking this in the Community. 

Appreciate your patience, I wanted to get the best information for you. You are able to link your Azure subscription (where you free credits live) to your account and use them toward some GitHub-metered products like Copilot Business, Actions overages, etc. 

For more information and instructions, please see [Connecting an Azure subscription](https://docs.github.com/billing/managing-the-plan-for-your-github-account/connecting-an-azure-subscription#title-h1) in the GitHub Docs.

**Comment by khakie** [comment]
Thank you so much for your detailed response! This is exactly the information I was looking for. I appreciate your help in clarifying the billing process and how to apply Azure free credits toward GitHub Copilot. I'll refer to the documentation you provided.

Thanks again, and I’ll go ahead and close this thread!

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
